pkg/aggregator/mocksender/asserts.go

```diff
@@ -43,29 +48,33 @@ func (m *MockSender) AssertMetricNotTaggedWith(t *testing.T, method string, metr
 	return m.Mock.AssertCalled(t, method, metric, mock.AnythingOfType("float64"), mock.AnythingOfType("string"), AssertTagsNotContains(t, tags))
 }
 
+// Return a bool value if all the elt of expected are inside the actual array
+func expectedInActual(actual, expected []string) bool {
```

Comment:
When reading the unit tests, I was surprised at why they'd work as the signature I was infering was `expectedInActual(expected, actual []string)`. Would it make sense to you to swap the arguments?

Reply:
I based the signature of this function on the original one we replaced:
`assert.Contains(t, actual, tag)` but I can swap the arguments of course.

Reply:
I'm expecting the first argument to `Contains` to be the one containing the second argument. I'm ok with this name too.

Reply:
I'll change it because it's more obvious in other languages too: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIn